src/main/java/org/opensearch/flowframework/workflow/RegisterLocalModelStep.java

```diff
@@ -198,4 +202,46 @@ public void onFailure(Exception e) {
     public String getName() {
         return NAME;
     }
+
+    /**
+     * Retryable get ml task
+     * @param workflowId the workflow id
+     * @param getMLTaskFuture the workflow step future
+     * @param taskId the ml task id
+     * @param retries the current number of request retries
+     */
+    void retryableGetMlTask(String workflowId, CompletableFuture<WorkflowData> registerLocalModelFuture, String taskId, int retries) {
```

Comment:
should we add some modifiable retry param as input to this step or as a setting in our plugin?

Reply:
We actually have a setting for maximum number of retries that the user can update. ([ref](https://github.com/opensearch-project/flow-framework/blob/772fbbd9eb58aeda52ea57a36307ad7206b1cf0b/src/main/java/org/opensearch/flowframework/common/FlowFrameworkSettings.java#L49))

This is passed to this step via the `AbstractRetryableWorkflowStep` class that we're extending here and used to check against the current retry count before attempting another retry